shared/router-v2/account-switcher/index.tsx

```diff
@@ -136,7 +136,7 @@ export default Kb.HeaderHoc(AccountSwitcher)
 const styles = Styles.styleSheetCreate(() => ({
   buttonBox: Styles.padding(0, Styles.globalMargins.small, Styles.globalMargins.tiny),
   desktopScrollview: {
-    width: 200,
+    width: '100%',
```

Comment:
Just curious. What that issue that needed full-width?

Reply:
Floating menus increased to 240px-wide.